README.md

```diff
@@ -12,6 +12,7 @@ Group 1
 6. Run "Npm install" and then "Npm run dev"
 7. Navigate to the vite localhost link
 8. Adde by me
+9- added in the f-branch
```

Comment:
Would you please add this line in the file X.md instead ?

Reply:
test

Reply:
for sure, I will apply